docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,77 @@ original:
 
 We (Student Robotics) manage the charging of all batteries at the competition. Battery Charging volunteers are responsible for ensuring that there is a continuous supply of charged batteries available for use by teams.
 
-## Equipment
-
- * Battery chargers
- * Discharged battery box
- * Charged battery box
- * Damaged battery/charger box
- * Red insulating tape
- * Example good batteries (one brand new, one squidgy)
+## Setup
+
+### Equipment
+Most batteries and chargers will arrive with teams, but any batteries, bags, and chargers in inventory should be prepared during setup.
+There are two locations involved in battery charging. The equipment required at each is as listed below:
+
+- Helpdesk
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x LiPo Bag for damaged batteries, clearly labelled, red taped
+    - At least one roll of red insulating tape
+- Battery Charging Station
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x Damaged battery box (18L RUB), clearly labelled, red taped
+    - At least one roll of red insulating tape
+    - A Radio for contacting the health and safety officer
+    - Example good batteries (one brand new, one squidgy)
+    - A copy of this document
+
+### Safe Use of Battery Charging Equipment - Battery Charging Station
+
+- There should be `0.5A` of mains electricity available per charger
+- It should be ensured that any power distribution is setup properly and able to handle the current draw
+- Additionally, there should be no trailing wires across the floor to prevent charging batteries falling on the floor
 
 ## Volunteer Requirements
 
-Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be discussed with Rich Barlow ASAP.
+Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be raised with helpdesk ASAP.
```

Comment:
Is helpdesk really the right place to raise this? Essentially the discussion I'd had on this previously was that issues should be raised within whoever trained the volunteer (as they're the person responsible for ensuring the volunteer can perform the task).

My other concern was that the people on helpdesk might not be able to help and would likely not know who to pass this on to, creating a dead-end.

Reply:
Okay I'll say whoever briefed you, failing that, the volunteer manager. Fixed in  5f667c2